src/components/Sum/Sum.jsx

```diff
@@ -1 +1,7 @@
-// export const Sum = () => ();
+export const Sum = ({ a = 0, b = 0 }) => (
```

Comment:
you already make export default in 7 line
Remove here or remove export default
```suggestion
const Sum = ({ a = 0, b = 0 }) => (
```